test/SafeAnchorSpec.js

```diff
@@ -72,18 +72,14 @@ describe('SafeAnchor', () => {
 
   it('Should disable link behavior', () => {
     let clickSpy = sinon.spy();
-    let spy = sinon.spy(SafeAnchor.prototype, 'handleClick');
 
     mount(
       <SafeAnchor disabled href="#foo" onClick={clickSpy}>
         Title
       </SafeAnchor>,
     ).simulate('click');
 
-    expect(spy).to.have.been.calledOnce;
     expect(clickSpy).to.have.not.been.called;
-    expect(spy.getCall(0).args[0].isDefaultPrevented()).to.equal(true);
```

Comment:
can we keep these asserts but make them on the `clickSpy`?

Reply:
@jquense I'm not sure if I'm following, since the whole point of this test was to see if `clickSpy` was called, no? How would these assertions pass if `clickSpy` was never called?

Reply:
oops yes sorry :P